express-delete/index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const arr = [];
+  for (var key in grades) {
+    arr.push(grades[key]);
+  }
+  res.json(arr);
+
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  res.send(req.params.id);
+  delete grades['47'];
+  res.sendStatus(404);
```

Comment:
This is all you need, this sends a response to the client with a specific status code. However we want to send a status code of `204` not `404`